torchaudio/datasets/librispeech.py

```diff
@@ -76,7 +76,74 @@ def load_librispeech_item(
     )
 
 
-class LIBRISPEECH(Dataset):
+def load_librispeech_item(
+    fileid: str, path: str, ext_audio: str, ext_txt: str
+) -> Tuple[Tensor, int, str, int, int, int]:
+    path = os.path.normpath(path)
+    root = os.path.dirname(path)
+    subset = os.path.basename(path)
+
+    filepath, sample_rate, transcript, speaker_id, chapter_id, utterance_id = get_librispeech_metadata(
+        fileid, root, subset, ext_audio, ext_txt
+    )
+    waveform, _ = torchaudio.load(os.path.join(root, filepath))
+
+    return (
+        waveform,
+        sample_rate,
+        transcript,
+        speaker_id,
+        chapter_id,
+        utterance_id,
+    )
+
+
+class LibriSpeechBase(Dataset):
+    """Create a Dataset for *LibriSpeech* [:footcite:`7178964`].
```

Comment:
I feel the description can be different from `LIBRISPEECH`, for example, explicitly mention the class is only for fetching metadata, without loading the actual waveforms.

Reply:
shall we also add it to the documentation page?

Reply:
I did not add it to docs yet since I wasn't sure the best way to do so with 2x API. Maybe something formatted like this?

```
+ LibriSpeech
   - LibriSpeechBase
   - LIBRISPEECH
```